Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM hydroshare/hs_docker_base:42e31f1
+FROM hydroshare/hs_docker_base:7a6c581
```

Comment:
I assume this image only has the change to Pillow version.

Reply:
correct